services/batch_manager.js

```diff
@@ -17,14 +17,17 @@ const sequelize = new Sequelize(dbConfig.database, dbConfig.username, dbConfig.p
 });
 
 exports.pre_process_params = async (req, resp) => {
-    if(req.query.type == "admin"){
+    if(req.query.type == "admin" && req.query.sports_id == null){
```

Comment:
are we passing sports_id filter from admin panel for listing batches?if not then we can remove && req.query.sports_id == null